how-to-guides/data-versioning/notebooks/Version_datasets_in_model_training_runs.ipynb

```diff
@@ -45,19 +45,16 @@
     ]
    },
    "source": [
-    "## Setup"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "tags": [
-     "comment",
-     "installation"
-    ]
-   },
-   "source": [
-    "Install dependencies"
+    "## Before you start\n",
+    "\n",
+    "Make sure that you have:\n",
+    "* [Python 3.7+ installed](https://www.python.org/downloads/),\n",
+    "* a [Neptune account](https://neptune.ai/register),\n",
+    "* [created a project](https://app.gitbook.com/@neptune-ai/s/docs-re-positioning/administration/workspace-project-and-user-management/projects#create-project) from the Neptune UI that you will use for tracking metadata.\n",
+    "\n",
+    "**Tip**\n",
+    "\n",
+    "Registering with Neptune and creating a project is optional in case you are just trying out the application as an \"ANONYMOUS\" user."
```

Comment:
Could you please replace this with the section from the new template? In particular, the GitBook link should not be there.